### PR TITLE
travis: automatically publish releases to pypi

### PR DESCRIPTION
**Pull request by ikonst**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,3 +35,17 @@ script:
...
+    - stage: deploy
+      python: '3.6'
+      deploy:
+        provider: pypi
+        user: lyft-pynamodb
+        password:
+          secure: "B2AAd62kRYOiCcLjhfIJsICS9laH9J9c5H4tU2C2Vojnb+3W/qj9xP+/GnEZ6Tc2h4SNMQ81Wtzn1CjD3Ut8CCudoBKSZnJmip+EACgpNXjZiU+g0PlEgVbDfJV15OrmFGUeDexxLF3DEQKp8YiMpAhJAH6+SV5W17Er1M6mwBs="
+        on:
+          tags: true
+          distributions: sdist
```
Comment (ikonst) [inline .travis.yml]: @garrettheel which ones did you upload? https://docs.travis-ci.com/user/deployment/pypi/#uploading-different-distributions

Comment (garrettheel) [reply .travis.yml]: I've just been running `publish`, which uses `sdist,bdist_wheel`. See  https://github.com/pynamodb/PynamoDB/blob/a1e9df6f3373665f1de64d35a556e63aefd43e3e/setup.py#L18-L23

Also I believe `distributions` should be under the `deploy` key directly, not `on`.

**Comment by ikonst** [comment]
@garrettheel better now?

**Comment by ikonst** [comment]
🚀

**Pull request merged**